src/components/Header/components/Language/index.jsx

```diff
@@ -1,15 +1,15 @@
 import { useState } from 'react'
```

Comment:
It would be nice to have the ability to change icon colors depending on the theme. Additionally, a default size is needed, along with the option to add a dynamic size. Maybe some other props, such as className, could also be added.

Think more about:
- Dynamic Color Management
Add support for multiple color props, such as:
fillColor – for fill color.
strokeColor – for stroke color.
This is especially useful for SVGs that support stroke.

- Icon Theming
Add theme support via context or a theme prop, allowing the icon to automatically adapt to themes (e.g., light/dark mode)